README_local.md

```diff
@@ -0,0 +1,16 @@
+## Running Locally
```

Comment:
whoops, ignore this, will revert (its for https://jira.mongodb.org/browse/REALMC-8828)